packages/components/src/visually-hidden/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * VisuallyHidden component to render text out non-visually
+ * for use in devices such as a screen reader.
+ */
+function VisuallyHidden( {
+	as = 'div',
+	...props
+} ) {
+	return renderAsRenderProps( {
+		as,
+		className: 'components-visually-hidden',
+		...props,
+	} );
+}
+
+// render "as" to be either string (tag name) or component (fn)
+function renderAsRenderProps( { as: T = 'div', ...props } ) {
```

Comment:
I like that pattern a lot and this can probably be extracted in a utils folder in the components package. That said a `<Component as="div" />` doesn't mean anything for mobile since there's no "div" on mobile and there's the expectation that the same props should be valid in all platform.

Not a blocker since mobile can decide to just ignore the "as"  prop but I thought I'd raise for awareness.

Reply:
I moved it to a `utils.js` file.

The div is just the default value, if used on Mobile they can pass in something different, or not use :-)

Reply:
The idea is that mobile and web use the exact same code in the `edit` function of blocks... So there's no mobile do that and web do this.

Reply:
@youknowriad ah, I see.  Sounds like its not a blocker though, so should be fine.
Is the utils.js extraction fine?

Reply:
Actually @gziolo has the change requested for the CSS. Greg, when you get the chance, can you do a quick review again. I think everything has been addressed. Thanks! :bowing_man: